partnerapp/src/Home.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+import { Button } from "reactstrap";
+
+import { CampaignStore } from "./campaign-store";
+import { observer } from "mobx-react-lite";
+
+export const HomeView: FC<{
+  store: CampaignStore;
+}> = observer(({ store }) => {
+  const onAddTokenClick = () => {
+    console.log("Add token");
```

Comment:
Is this where they should add a hardcoded token? Should we in that case have a good way of setting the token here? This doesn't really describe how to do that..?